docs/conmon.8.md

```diff
@@ -0,0 +1,140 @@
+conmon 8 "User Commands"
+==================================================
+
+# NAME
+
+conmon - container monitor utility
+
+# SYNOPSIS
+
+conmon [options]
+
+# DESCRIPTION
+
+conmon is a command line program for monitorying and managing the lifecycle of
+linux containers follow the Open Container Initiative (OCI) format.
+
+# APPLICATION OPTIONS
+
+**--api-version**
+Conmon API version to use
+
+**-b**, **--bundle**
+Bundle path
+
+**-c**, **--cid**
+Container ID
+
+**--exec-attach**
+Attach to an exec session
+
+**-e**, **--exec**
+Exec a command in a running container
+
+**--exec-process-spec**
+Path to the process spec for exec
+
+**--exit-command**
+Path to the program to execute when the container terminates its execution
+
+**--exit-command-arg**
+Additional arg to pass to the exit command.  Can be specified multiple times
+
+**--exit-delay**
+Delay before invoking the exit command (in seconds)
+
+**--exit-dir**
+Path to the directory where exit files are written
+
+**-h**, **--help**
+Show help options
+
+**-i**, **--stdin**
+stdin
+
+**-l**, **--log-path**
+Log file path
+
+**--leave-stdin-open**
+Leave stdin open when attached client disconnects
+
+**--log-level**
+Print debug logs based on log level
+
+**--log-size-max**
+Maximum size of log file
+
+**--log-tag**
+Additional tag to use for logging
+
+**-n**, **--name**
+Container name
+
+**--no-new-keyring**
+Do not create a new session keyring for the container
+
+**--no-pivot**
+Do not use pivot_root
+
+**--no-sync-log**
+Do not manually call sync on logs after container shutdown
+
+**-0**, **--persist-dir**
+Persistent directory for a container that can be used for storing container data
+
+**--pidfile**
+PID file (DEPRECATED)
+
+**-p**, **--container-pidfile**
+Container PID file
+
+**-P**, **--conmon-pidfile**
+Conmon daemon PID file
+
+**-r**, **--runtime**
+Runtime path
+
+**--replace-listen-pid**
+Replace listen pid if set for oci-runtime pid
+
+**--restore**
+Restore a container from a checkpoint
+
+**--restore-arg**
+Additional arg to pass to the restore command. Can be specified multiple times. (DEPRECATED)
+
+**--runtime-arg**
+Additional arg to pass to the runtime. Can be specified multiple times
+
+**--runtime-opt**
+Additional opts to pass to the restore or exec command. Can be specified multiple times
+
+**-s**, **--systemd-cgroup**
+Enable systemd cgroup manager
+
+**--socket-dir-path**
+Location of container attach sockets
+
+**--sync**
+Allowing caller to keep the main conmon process as its child by only forking once
+
+**--syslog**
+Log to syslog (use with cgroupfs cgroup manager)
+
+**-t**, **--terminal**
+Terminal
```

Comment:
can we expound a bit more on this?

Reply:
Sure I have no idea what this means.  @haircommander Below there is another field on hold open standard IO?

Does this mean the same thing as podman run -t?

```
       Allocate a pseudo-TTY. The default is false.

       When set to true, Podman will allocate a pseudo-tty and attach to the standard input of the container. This can be used, for example, to run a throwaway interactive shell. The default is false.

       NOTE: The -t option is incompatible with a redirection of the Podman client standard input.
```

Reply:
yeah conmon has a `-i` that corresponds to podman's stdin.

`-t` should be called when the manager wants a terminal connection. Conmon will open a pseudoterminal console, and pass it to the runtime with `--console-socket`